product_docs/docs/biganimal/release/knowledge_base.mdx

```diff
@@ -18,3 +20,9 @@ See the following articles for step-by-step instructions for setting up specific
 - [Using Google Workspace (G Suite) as your identity provider](https://support.biganimal.com/hc/en-us/articles/17019580767385-Steps-to-Configure-Google-Workspace-G-Suite-as-your-identity-provider)
 - [Using Okta as your identity provider](https://support.biganimal.com/hc/en-us/articles/17019836140441-Steps-to-Configure-Okta-as-your-identity-provider)
 
+## Configuring connection poolers
+
+See the following articles for step-by-step instructions for setting up connection poolers:
+
+- [How to configure PgBouncer with BigAnimal Cluster](https://support.biganimal.com/hc/en-us/articles/4848726654745-How-to-configure-PgBouncer-with-BigAnimal-Cluster
+- [How to configure Pgpool with a BigAnimal cluster](https://support.biganimal.com/hc/en-us/articles/5475273722009-How-to-configure-Pgpool-with-BigAnimal-Cluster)
```

Comment:
Are both of the above links to setting up self-hosted pgbouncer or pgpool with BigAnimal? Because for BigAnimal-hosted pooling, I'm 99% certain pgbouncer is the only option, and also, setting it up would go through our web UI and therefore be very different from how to set up self-hosted pgbouncer.